src/runtime_src/core/tools/xbutil2/npu-smi

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2020-2022 Xilinx, Inc.
+#
+
+# -- Detect a Windows environment and automatically switch to the .bat file
+if [[ "`uname`" == windows32* ]] || [[ "`uname`" == CYGWIN* ]] ; then
+    trap "" INT
+    "$0.bat" "$@"
+    exit $?
+fi
+
+# Working variables
+XRT_PROG=xbutil2
+
+# -- Examine the options
+XRTWARP_PROG_ARGS_size=0
+XRTWRAP_PROG_ARGS=()
+while [ $# -gt 0 ]; do
+    case "$1" in
+    # Indicates that the legacy was specified
+    -legacy|--legacy)
+        echo "----------------------------------------------------------------------"
+        echo "Error: Obsoleted option
+       The --legacy option is no longer supported.
+    
+       For legacy migration, please refer to the XRT github.io utility 
+       migration guide.
+
+       Commands, options, arguments and their descriptions can also be 
+       reported via the --help option."
+        echo "----------------------------------------------------------------------"
```

Comment:
We don't need this anymore, but is there a way we can do something like this for "xbutil"? So if someone uses xbutil, we should inform them that this is a deprecated toolname, please use "npu-smi